changelogs/unreleased/1930-nrb

```diff
@@ -0,0 +1 @@
+Add a new required --plugins flag for velero install command. --plugins takes a list of container images to add as initcontainers.
```

Comment:
Technically it's optional now, but will be required by the time we release - I'm fine leaving it as-is given that it'll change, but wasn't sure if it was intentional or not.

Reply:
I'll change it to required when I merge the cloud provider extraction PR. I added a todo for that.